### PR TITLE
[Merged by Bors] - [DOC] Mention FPS in log_diagnostics example

### LINKED ISSUE #2076: Note that the log_diagnostics example shows how to check FPS in description
## How can Bevy's documentation be improved?

For discoverability from users searching, this should mention that it shows the "frames per second (FPS)"

### PR DESCRIPTION
**Pull request by FlyingRatBull**
Closes #2076

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ Example | File | Description
 Example | File | Description
 --- | --- | ---
 `custom_diagnostic` | [`diagnostics/custom_diagnostic.rs`](./diagnostics/custom_diagnostic.rs) | Shows how to create a custom diagnostic
-`log_diagnostics` | [`diagnostics/log_diagnostics.rs`](./diagnostics/log_diagnostics.rs) | Add a plugin that logs diagnostics to the console
+`log_diagnostics` | [`diagnostics/log_diagnostics.rs`](./diagnostics/log_diagnostics.rs) | Add a plugin that logs FPS / frame time diagnostics to the console
```
Comment (NathanSWard) [inline examples/README.md:145]:
```suggestion
`log_diagnostics` | [`diagnostics/log_diagnostics.rs`](./diagnostics/log_diagnostics.rs) | Add a plugin that logs FPS, frame time, and frame count diagnostics to the console
```

Comment (NathanSWard) [inline examples/README.md:145]:
```suggestion
`log_diagnostics` | [`diagnostics/log_diagnostics.rs`](./diagnostics/log_diagnostics.rs) | Add a plugin that logs FPS, frame time, and frame count diagnostics to the console
```

Comment (FlyingRatBull) [reply examples/README.md:145]: I thought about that but discarded it because of brevity :D

**Comment by FlyingRatBull** [comment]
> FPS, frame time, and frame count
Do you think FPS is enough to be found by all interested users, or should it be something like `frames per second (FPS)` as mentioned in #2076 ?

**Comment by NathanSWard** [comment]
> Do you think FPS is enough to be found by all interested users, or should it be something like frames per second (FPS) as mentioned in #2076 ?

Um now that I think of it, while _most_ game devs (I assume) would know what FPS stands for, that doesn't mean everyone will.
I would agree that we should change `FPS` to `frames per second (FPS)`

**Comment by FlyingRatBull** [comment]
CI seems to have found an [unrelated issue](https://github.com/bevyengine/bevy/pull/2150/checks?check_run_id=2567817136)

File: examples/README.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ Example | File | Description
 Example | File | Description
 --- | --- | ---
 `custom_diagnostic` | [`diagnostics/custom_diagnostic.rs`](./diagnostics/custom_diagnostic.rs) | Shows how to create a custom diagnostic
-`log_diagnostics` | [`diagnostics/log_diagnostics.rs`](./diagnostics/log_diagnostics.rs) | Add a plugin that logs diagnostics to the console
+`log_diagnostics` | [`diagnostics/log_diagnostics.rs`](./diagnostics/log_diagnostics.rs) | Add a plugin that logs frames per second (FPS), frame time, and frame count diagnostics to the console
```
Comment (mockersf) [inline examples/README.md:145]:
```suggestion
`log_diagnostics` | [`diagnostics/log_diagnostics.rs`](./diagnostics/log_diagnostics.rs) | Add a plugin that logs diagnostics, like frames per second (FPS), to the console
```
Diagnostics are not limited to those, we shouldn't list them as if it was exhaustive

**Comment by mockersf** [comment]
> CI seems to have found an [unrelated issue](https://github.com/bevyengine/bevy/pull/2150/checks?check_run_id=2567817136)

should be fixed now, `notify` released a new version that fix their dependency

File: examples/README.md (hunk 2 above)
Comment (FlyingRatBull) [reply examples/README.md:145]: Good point indeed

**Review by NathanSWard — APPROVED** [review]

**Review by mockersf — APPROVED** [review]

**Pull request closed without merging**